.github/scripts/mega-app-copy-files.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+cd build-system-tests
+if [ "$BUILD_TOOL" == 'cra' ];then
+    cp templates/template-aws-exports.js mega-apps/${MEGA_APP_NAME}/src/aws-exports.js
+    if [ "$LANGUAGE" == 'js' ];then
+        cp templates/components/react/cra/App.js mega-apps/${MEGA_APP_NAME}/src/App.js
+    else
+        cp templates/components/react/cra/App.js mega-apps/${MEGA_APP_NAME}/src/App.tsx
+        if [ "$FRAMEWORK_VERSION" == '16' ];then
+            cp templates/components/react/cra/index-react-${FRAMEWORK_VERSION}.js mega-apps/${MEGA_APP_NAME}/src/index.tsx
```

Comment:
Can you put a comment why we copy this file only for React 16?

Reply:
Done 3b8c87120cb30bf33034079b7895aff4854db685